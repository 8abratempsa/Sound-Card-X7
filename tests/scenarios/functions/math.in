octosql "SELECT abs(42), abs(-42), abs(42.0), abs(-42.0),
                sqrt(49.0),
                ceil(42.0), ceil(42.5),
                floor(42.0), floor(42.5),
                log2(8.0),
                log(9.0),
                log10(100.0),
                pow(2.0, 3.0)"
