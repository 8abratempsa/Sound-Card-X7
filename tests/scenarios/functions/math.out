+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------------------+--------+--------+
| col_0 | col_1 | col_2 | col_3 | col_4 | col_5 | col_6 | col_7 | col_8 | col_9 |       col_10       | col_11 | col_12 |
+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------------------+--------+--------+
|    42 |    42 |    42 |    42 |     7 |    42 |    43 |    42 |    42 |     3 | 2.1972245773362196 |      2 |      8 |
+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+--------------------+--------+--------+
